rest-service/manager_rest/rest/resources_v3_1/deployments.py

```diff
@@ -212,9 +212,9 @@ def put(self):
         return sm.put(deployment_dependency)
 
     @staticmethod
-    def _verify_and_get_source_and_target_deployments(sm,
-                                                      source_deployment_id,
-                                                      target_deployment_id):
+    def _verify_and_get_source_and_target_deployments(
+            sm, source_deployment_id, target_deployment_id,
+            is_component_deletion=False):
```

Comment:
Just in terms of styling, it would be great if you could make it look like a nice list (like the one before), I guess flake8 is the problem but there might be a solution :)